tensorflow_addons/image/distort_image_ops_test.py

```diff
@@ -111,6 +111,23 @@ def test_invalid_shapes(self):
                                       "but instead has 4 channels"):
             self.evaluate(self._adjust_hue_in_yiq_tf(x_np, delta_h))
 
+    def test_adjust_hsv_in_yiq_unknown_shape(self):
+        fn = distort_image_ops.adjust_hsv_in_yiq.get_concrete_function(
+            tf.TensorSpec(shape=None, dtype=tf.double))
```

Comment:
Maybe float32 is enough?

Reply:
Good catch! As `image_np = np.random.rand(*shape) * 255.` is `float64`, the dtype here is changed to `tf.float64`.